lrpar/src/lib/ctbuilder.rs

```diff
@@ -256,6 +235,13 @@ where
         self
     }
 
+    pub fn conflicts(&self) -> Option<(&YaccGrammar<StorageT>, &StateGraph<StorageT>, &StateTable<StorageT>, &Conflicts<StorageT>)> {
+        if let Some((ref grm, ref sgraph, ref stable)) = self.cdata {
```

Comment:
Can you simplify this to `if let Some((grm, sgraph, stable)) = &self.cdata`? I'm not sure.

Reply:
Looks like I can! :-)